gnomon/db/migrations/001_polyphemus_db_init.rb

```diff
@@ -0,0 +1,10 @@
+Sequel.migration do 
```

Comment:
hm, do these migrations still apply?

Reply:
Err, just got lazy cleaning up stuff and started making the front-end. I'll clean this up when I get to the backend stuff.